packages/multichain-network-controller/src/constants.ts

```diff
@@ -0,0 +1,78 @@
+import { type StateMetadata } from '@metamask/base-controller';
+import { BtcScope, SolScope } from '@metamask/keyring-api';
+import { NetworkStatus } from '@metamask/network-controller';
+
+import type {
+  MultichainNetworkConfiguration,
+  MultichainNetworkControllerState,
+  MultichainNetworkMetadata,
+  SupportedCaipChainId,
+} from './types';
+
+export const BTC_NATIVE_ASSET = `${BtcScope.Mainnet}/slip44:0`;
+export const SOL_NATIVE_ASSET = `${SolScope.Mainnet}/token:EPjFWdd5AufqSSqeM2qN1xzybapC8G4wEGGkZwyTDt1v`;
+
+/**
+ * Supported networks by the MultichainNetworkController
+ */
+export const AVAILABLE_MULTICHAIN_NETWORK_CONFIGURATIONS: Record<
+  SupportedCaipChainId,
+  MultichainNetworkConfiguration
+> = {
+  [BtcScope.Mainnet]: {
+    chainId: BtcScope.Mainnet,
+    name: 'Bitcoin Mainnet',
+    blockExplorerUrls: ['https://blockstream.info'],
+    defaultBlockExplorerUrlIndex: 0,
+    nativeCurrency: BTC_NATIVE_ASSET,
+    isEvm: false,
+  },
+  [SolScope.Mainnet]: {
+    chainId: SolScope.Mainnet,
+    name: 'Solana Mainnet',
+    blockExplorerUrls: ['https://explorer.solana.com'],
+    defaultBlockExplorerUrlIndex: 0,
+    nativeCurrency: SOL_NATIVE_ASSET,
+    isEvm: false,
+  },
+};
+
+/**
+ * Metadata for the supported networks.
+ */
+export const NETWORKS_METADATA: Record<string, MultichainNetworkMetadata> = {
+  [BtcScope.Mainnet]: {
+    features: [],
+    status: NetworkStatus.Available,
+  },
+  [SolScope.Mainnet]: {
+    features: [],
+    status: NetworkStatus.Available,
+  },
+};
+
+/**
+ * Default state of the {@link MultichainNetworkController}.
+ *
+ * @returns The default state of the {@link MultichainNetworkController}.
+ */
+export const DEFAULT_MULTICHAIN_NETWORK_CONTROLLER_STATE: MultichainNetworkControllerState =
```

Comment:
Nit: Not a big deal, but it would be good if this were a function rather than a constant to align with other controllers, see here for more: https://github.com/MetaMask/core/blob/main/docs/writing-controllers.md#provide-a-default-representation-of-state